internal/experiment/webconnectivityqa/badssl.go

```diff
@@ -43,8 +43,8 @@ func badSSLWithWrongServerName() *TestCase {
 			HTTPExperimentFailure: "ssl_invalid_hostname",
 			XStatus:               16, // StatusAnomalyControlFailure
 			XNullNullFlags:        4,  // analysisFlagNullNullTLSMisconfigured
-			Accessible:            nil,
-			Blocking:              nil,
+			Accessible:            false,
+			Blocking:              false,
```

Comment:
This is an improvement rather than a regression. Web Connectivity LTE correctly flags the website as being down because of misconfigured TLS, while Web Connectivity v0.4 is not able to do that.